modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -0,0 +1,215 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-user-namespaces.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-pods-user-namespaces-configuring_{context}"]
+= Configuring Linux user namespace support
+
+
+.Prerequisites
+
+* You enabled the required Technology Preview features for your cluster by editing the `FeatureGate` CR named `cluster`:
++
+[source,terminal]
+----
+$ oc edit featuregate cluster
+----
++
+.Example `FeatureGate` CR
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: TechPreviewNoUpgrade <1>
+----
+<1> Enables the required `UserNamespacesSupport` and `ProcMountType` features.
++
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. This feature set allows you to enable these Technology Preview features on test clusters, where you can fully test them. Do not enable this feature set on production clusters.
+====
++
+After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled while the change is being applied.
+
+* You enabled the crun container runtime on the worker nodes. crun is currently the only released OCI runtime with support for user namespaces.
```

Comment:
```suggestion
* You enabled the `crun` container runtime on the worker nodes. `crun` is currently the only released OCI runtime with support for user namespaces.
```

Reply:
crun is the name of the runtime and doesn't take tick, the same as any other third-party package we use.